capstone-frontend/src/components/ClubSearchList.jsx

```diff
@@ -66,8 +66,7 @@ export class ClubSearchList extends Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.searchQuery !== prevProps.searchQuery
-        || this.props.books !== prevProps.books) {
+    if (this.props.books !== prevProps.books) {
```

Comment:
Why this change?

Reply:
With the original check, I kept running into the issue where this ClubSearchList page would reload without the new list of books being properly populated. It would render the page twice, once when the searchQuery changed and once when the fetches to the list of searched books finished.

For some reason, whenever I searched for a club then entered a query that shouldn't have resulted in any clubs, the list of clubs on this page stayed the same from the previous query.

This new check ensures that the page doesn't render multiple times and generates the correct clubs.

Reply:
Got it, nice catch!